in_toto/verifylib.py

```diff
@@ -39,19 +39,61 @@
 import in_toto.models.layout
 import in_toto.models.link
 import in_toto.formats
-from in_toto.models.metadata import Metablock
+from in_toto.models.metadata import Metadata
 from in_toto.exceptions import (RuleVerificationError, LayoutExpiredError,
-    ThresholdVerificationError, BadReturnValueError,
-    SignatureVerificationError)
-from securesystemslib.gpg.exceptions import KeyExpirationError
+    ThresholdVerificationError, BadReturnValueError)
 import in_toto.rulelib
 
+import securesystemslib.formats
+from securesystemslib.exceptions import VerificationError
+from securesystemslib.gpg.exceptions import KeyExpirationError
+
 # Inherits from in_toto base logger (c.f. in_toto.log)
 LOG = logging.getLogger(__name__)
 
 RULE_TRACE = {}
 
 
+def verify_metadata_signatures(metadata: Metadata, keys_dict):
```

Comment:
For the sake of a smaller diff you could just keep the original `verify_layout_signatures` and only update docs and variable names to  `s/metablock/metadata`, right?